src/frontend/src/expr/mod.rs

```diff
@@ -253,18 +253,6 @@ impl From<Condition> for ExprImpl {
     }
 }
 
-impl<'a> From<&'a Condition> for ExprImpl {
-    // TODO(TaoWu): We might also use `Vec<ExprImpl>` form of predicates in compute node,
```

Comment:
Should the TODO be kept?

Reply:
A little confused about why it is here, maybe we can create an issue for that? @neverchanje

Reply:
I think it's OK to be removed since it seems not urgent or critical. BTW, in general I prefer tracking issues to TODOs in code.

cc @neverchanje